fe/fe-core/src/main/java/com/starrocks/sql/analyzer/AlterTableStatementAnalyzer.java

```diff
@@ -292,6 +292,10 @@ public Void visitAddColumnClause(AddColumnClause clause, ConnectContext context)
                 throw new SemanticException(PARSER_ERROR_MSG.invalidColumnDef(e.getMessage()), columnDef.getPos());
             }
 
+            if (columnDef.getType().isTime()) {
```

Comment:
create table also need this check.

Reply:
create table have been checked in the current code. It just miss for DDL